crates/rolldown/src/ecmascript/ecma_generator.rs

```diff
@@ -48,10 +48,15 @@ impl Generator for EcmaGenerator {
       })
       .collect::<Vec<_>>();
 
-    rendered_module_sources.iter().for_each(|(_, module_id, _)| {
+    rendered_module_sources.iter().for_each(|(_, module_id, sources)| {
       // FIXME: NAPI-RS used CStr under the hood, so it can't handle null byte in the string.
       if !module_id.starts_with('\0') {
-        rendered_modules.insert(module_id.clone(), RenderedModule { code: None });
+        let rendered_length = sources.as_ref().and_then(|sources| {
+          sources
+            .get(1)
+            .and_then(|source| u32::try_from(source.content().encode_utf16().count()).ok())
+        });
+        rendered_modules.insert(module_id.clone(), RenderedModule { code: None, rendered_length });
```

Comment:
If code is return to the JS side, we probably should be able to just call `this.code.length` instead of calculating it in rust side.

Reply:
Do you mean we could discard the `renderedLength` option and simply return `code`, allowing users to call `this.code.length` on the js side instead?